client/src/components/ProfileListings/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router';
+import { Box, Paper, Typography } from '@material-ui/core';
+import Location from '@material-ui/icons/LocationOn';
+import Rating from '@material-ui/lab/Rating';
+import { Profile } from '../../../interface/Profile';
+import useStyles from './useStyles';
+
+interface Props {
+  profile: Profile;
+}
+export default function ProfileCard({ profile }: Props): JSX.Element {
+  const classes = useStyles();
+  const [isMouseOver, setIsMouseOver] = useState(false);
+  const history = useHistory();
+
+  const { address, _id: id, profilePic, introduction, pitch, rate, firstName, lastName } = profile;
+
+  const handleCardSelect = () => history.push(`/profile/${id}`);
+
+  return (
+    <Paper
+      component="article"
+      onMouseOver={() => setIsMouseOver(true)}
+      onMouseOut={() => setIsMouseOver(false)}
+      onDoubleClick={handleCardSelect}
+      className={classes.paper}
+      key={id}
+      elevation={isMouseOver ? 20 : 5}
+    >
+      <Box className={classes.cardBody}>
```

Comment:
Suggestion: It would be better to use the MUI Card API as each profile card contains information about a single person/topic.

Apart from adding semantic meaning, Cards support support multiple actions, UI controls, and an overflow menus.

For eg: The action area of the card can be specified by wrapping its contents in a `CardActionArea` component, which adds a hover effect on mouse enter/exit


Ref: https://mui.com/api/card/